src/lib.rs

```diff
@@ -101,7 +132,7 @@ fn is_header_name_token(b: u8) -> bool {
 }
 
 static HEADER_VALUE_MAP: [bool; 256] = byte_map![
-    0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0,
+    0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0,
```

Comment:
Yikes! Just checked, this was previously allowing Backspace instead of Horizontal Tab 😢 

Thanks for the fix!